src/backupNg/backupNg.spec.js

```diff
@@ -66,4 +66,42 @@ describe("backupNg", () => {
       expect(schedule.jobId).toBe(jobId);
     });
   });
+
+  describe(".delete() :", () => {
+    it("deletes a backup job", async () => {
+      let scheduleId = "scheduleTempId";
+      const { id: jobId } = await xo.call("backupNg.createJob", {
+        ...defaultBackupNg,
+        schedules: {
+          [scheduleId]: {
+            name: "scheduleTest",
+            cron: "0 * * * * *",
+          },
+        },
+        settings: {
+          ...defaultBackupNg.settings,
+          [scheduleId]: { snapshotRetention: 1 },
+        },
+      });
+      const backupNgJob = await xo.call("backupNg.getJob", { id: jobId });
+      for (const key in backupNgJob.settings) {
+        if (key !== "") {
+          expect(backupNgJob.settings[key]).toEqual({ snapshotRetention: 1 });
```

Comment:
We don't want to test the job creation but we want to get the schedule id. Please use `lodash/findkey` instead.